spec/models/miq_report_spec.rb

```diff
@@ -40,6 +40,34 @@
 end
 
 describe MiqReport do
+  context ".for_user" do
+    let(:my_user) { FactoryGirl.create(:user_with_group) }
+    let(:group_in_my_tenant) { FactoryGirl.create(:miq_group, :tenant => my_user.current_tenant) }
+
+    let(:other_tenant) { FactoryGirl.create(:tenant) }
+    let(:group_in_other_tenant) { FactoryGirl.create(:miq_group, :tenant => other_tenant) }
+
+    let(:my_report) { FactoryGirl.create(:miq_report, :miq_group => my_user.current_group, :rpt_type => "Custom") }
+    let(:report_in_my_tenant) { FactoryGirl.create(:miq_report, :miq_group => group_in_my_tenant, :rpt_type => "Custom") }
+    let(:report_in_another_tenant) { FactoryGirl.create(:miq_report, :miq_group => group_in_other_tenant, :rpt_type => "Custom") }
+
+    it "returns reports created by me or anyone in a group in my tenant" do
+      my_report
+      report_in_my_tenant
+      report_in_another_tenant
+      User.current_user = my_user
+
+      expect(described_class.for_user(my_user)).to match_array([my_report, report_in_my_tenant])
+    end
+
+    it "does not return reports created by a group in another tenant" do
```

Comment:
I think you can just remove this spec.

By asserting that the array only contains `my_report` and `report_in_my_tenant` in the first spec you're also asserting that `report_in_another_tenant` was not returned even though you created it.

Reply:
Also done 👍